tests/metrics/functional/test_classification.py

```diff
@@ -187,6 +187,9 @@ def test_confusion_matrix():
     cm = confusion_matrix(pred, target, normalize=False, num_classes=3)
     assert torch.allclose(cm, torch.tensor([[5., 0., 0.], [0., 0., 0.], [0., 0., 0.]]))
 
+    cm = confusion_matrix(pred, target, normalize=True, num_classes=3)
+    assert torch.allclose(cm, torch.tensor([[1., 0., 0.], [0., 0., 0.], [0., 0., 0.]]))
```

Comment:
None of these test seems to actually capture that the normalization was done incorrect. Could you change the examples to something like:
 https://scikit-learn.org/stable/auto_examples/model_selection/plot_confusion_matrix.html

Reply:
Implemented test directly from https://scikit-learn.org/stable/auto_examples/model_selection/plot_confusion_matrix.html at 31d22bc